public/scripts/frames.js

```diff
@@ -125,7 +127,15 @@ function setDataIntoCard() {
 
     });
 
+    return "ok";
+}
 
+function fillLogWork() {
```

Comment:
This will receive everything like title, estimatedTime, logWorkTotalTime, workDone and Id (TaskIdString)

Reply:
ok